xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -1059,6 +1104,21 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Whether equipment is suitable for cycles.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="RestStopDistance" type="xsd:decimal" minOccurs="0">
```

Comment:
Is there a reason why this is not a LengthType?

Reply:
no ...

Reply:
done